### PR TITLE
tpl: Add EnableMissingTranslationPlaceholders option

### LINKED ISSUE #2451: i18n: return empty string by default when no match
If `i18n` finds no matches, it should return a nil value (empty string?) instead of the string "[i18n] %s". 

It could also simply return the identifier, but a nil value is idiomatic go and would let us check if  a match exists in template code, e.g. `{{ with i18n "some_identifier" }}`.

### PR DESCRIPTION
**Pull request by digitalcraftsman**
Fixes #2451

### CODE REVIEW

**Comment by moorereason** [comment]
You need to `gofmt` `tpl/template_i18n.go`.

File: tpl/template_i18n.go [hunk 1]
```diff
@@ -82,6 +82,10 @@ func SetI18nTfuncs(bndl *bundle.Bundle) {
 					return translated
 				}
 			}
+
+			if(viper.GetBool("EnableMissingTranslationPlaceholders")) {
```
Comment (bep) [inline tpl/template_i18n.go]: You will have to add this conditional above, too.

Comment (bep) [inline tpl/template_i18n.go]: if not ...

Comment (digitalcraftsman) [inline tpl/template_i18n.go]: Done

**Comment by digitalcraftsman** [comment]
`gofmt` - done.

**Comment by mitchchn** [comment]
Thank you @digitalcraftsman!

**Pull request merged**